frontend/admin/src/js/components/pool/CreatePool.tsx

```diff
@@ -244,13 +244,12 @@ export const CreatePoolForm: React.FunctionComponent<CreatePoolFormProps> = ({
                 description:
                   "Placeholder displayed on the pool form classifications field.",
               })}
-              name="classifications"
               options={classificationOptions}
               rules={{
                 required: intl.formatMessage(errorMessages.required),
               }}
```

Comment:
Re: https://github.com/GCTC-NTGC/gc-digital-talent/pull/3536

If the above gets merged first, we can do up to 150 search-replaces like this and ditch some boilerplate:

```suggestion
              rules={{ required: true }}
```